### PR TITLE
 Removed lmax disruptor

### LINKED ISSUE #175: Remove com.lmax:disruptor
**Describe the bug**
As we are not using Async Appender and even if we are using, there is no requirement for using it so we can remove com.lmax:disruptor dependency from pom and build.gradle.

**Expected behavior**
Webapplication should be light weight and the dependencies not required should be removed.

**Additional context**
You can also help us find any other such dependency which we can remove and make the project better.

### PR DESCRIPTION
**Pull request by hemantgs**
Fixes #175 
* Removed lmax disruptor
* Changed log level from ALL to INFO
* Did away with async logger

Fixed wrong commit

### CODE REVIEW

File: src/main/resources/log4j2.xml [hunk 1]
```diff
@@ -40,11 +40,7 @@
 
 
 	<Loggers>
-		<AsyncLogger name="org.sasanlabs" level="ERROR">
-			<AppenderRef ref="CONSOLE" />
-			<AppenderRef ref="ROLLING_LOG_FILE" />
-		</AsyncLogger>
-		<Root level="ALL">
+		<Root level="INFO">
```
[the remark this answers is not in the document]
Comment (hemantgs) [reply src/main/resources/log4j2.xml]: Done

File: build.gradle [hunk 2]
```diff
@@ -60,8 +60,6 @@ dependencies {
     implementation group: 'org.apache.commons', name: 'commons-text', version: '1.8'
 	//Log4j Dependencies
     implementation group: 'org.springframework.boot', name: 'spring-boot-starter-log4j2'
-  	// for Async Logging. Used by Log4j2
-    implementation group: 'com.lmax', name: 'disruptor', version: '3.4.0'
```
[the remark this answers is not in the document]
Comment (preetkaran20) [reply build.gradle:64]: I added both when i started, not sure why but actually we should now start depending only on build.gradle. -> this needs some documentation, github action and release notes update. Let's do it the next time we need to update any dependency.

**Comment by preetkaran20** [comment]
LGTM

**Comment by preetkaran20** [comment]
@hemantgs  thanks a lot for the contribution.

**Pull request merged**